dockerfiles/cli/cli-dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.18.5
+FROM harbor.galasa.dev/docker_proxy_cache/alpine:3.18.5
```

Comment:
I believe this will need 'library' so harbor.galasa.dev/docker_proxy_cache/library/alpine:3.18.5 - same for the rest of the alpine ones if you could make that change please :)